packages/blade/src/components/Input/BaseInput/baseInputStyles.ts

```diff
@@ -46,32 +46,33 @@ export const getInputBackgroundAndBorderStyles = ({
   | 'isDropdownTrigger'
 >): CSSObject => {
   // normal state
-  let backgroundColor = theme.colors.brand.gray.a50.lowContrast;
-  let borderBottomColor = theme.colors.brand.gray.a100.lowContrast;
+  let backgroundColor = theme.colors.interactive.background.gray.default;
+  let borderBottomColor = theme.colors.interactive.background.gray.highlighted; // TODO: Check why this is not a border color
 
   // hoverState
   if (isHovered) {
-    backgroundColor = theme.colors.brand.gray.a100.lowContrast;
+    backgroundColor = theme.colors.interactive.background.gray.highlighted;
   }
 
   // focused state
   if (isFocused) {
-    backgroundColor = theme.colors.brand.primary[300];
+    backgroundColor = theme.colors.interactive.background.primary.faded;
+    borderBottomColor = theme.colors.interactive.background.primary.default; // TODO: Check why this is not a border color
```

Comment:
Checked with design, they're using a background color here since its an animated block and not really a border